StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/OpenAuthenticatedWebViewMessageHandler.swift

```diff
@@ -8,14 +8,20 @@
 import Foundation
 
 /// Indicates to open the provided URL in an `ASWebAuthenticationSession`.
-class OpenAuthenticatedWebViewMessageHandler: ScriptMessageHandler<OpenAuthenticatedWebViewMessageHandler.Payload> {
+class OpenAuthenticatedWebViewMessageHandler: ScriptMessageHandlerWithReply<OpenAuthenticatedWebViewMessageHandler.Payload, OpenAuthenticatedWebViewMessageHandler.Response> {
```

Comment:
Proposing that we combine `OpenAuthenticatedWebViewMessageHandler` and `ReturnedFromAuthenticatedWebViewSender` into a single async handler.

See [slack discussion](https://stripe.slack.com/archives/C06JRADKJF7/p1729033305657649?thread_ts=1729031087.701449&cid=C06JRADKJF7)

Reply:
Opted not to do this since Android cannot support this